checkout/application/orderService.go

```diff
@@ -62,11 +62,59 @@ const (
 	LastPlacedOrderSessionKey = "orderservice_last_placed"
 )
 
-var orderFailedStat = stats.Int64("flamingo-commerce/orderfailed", "my stat records 1 occurrence per error", stats.UnitDimensionless)
+var (
+	// orderValidationFailCount counts validation failures on orders
+	orderValidationFailCount = stats.Int64("flamingo-commerce/checkout/order_validation_failed", "Count of failures while validating orders", stats.UnitDimensionless)
+
+	// decoratedCartCreationFailCount counts creation errors for decorated carts
+	decoratedCartCreationFailCount = stats.Int64("flamingo-commerce/checkout/decorated_cart_creation_failed", "Count of failures while creating decorated cart", stats.UnitDimensionless)
```

Comment:
I guess we can remove that one since it's not really related to the order service, that's something we could add to the cart service in the future.

Reply:
Removed it :)